apps/services/personal-representative/infra/personal-representative.ts

```diff
@@ -0,0 +1,40 @@
+import { ref, service, ServiceBuilder } from '../../../../infra/src/dsl/dsl'
+
+export const serviceSetup = (): ServiceBuilder<'personal-representative'> => {
+  return service('personal-representative')
```

Comment:
I don't see any secrets defined for any of these services. Are they missing?

Reply:
I'm basing this off charts in https://github.com/island-is/helm/tree/main/charts/identity-server

I noticed there are a couple of charts that don't seem to use secrets.

Are there supposed to be secrets on these?